examples/chip-tool/commands/clusters/ClusterCommand.h

```diff
@@ -68,8 +69,9 @@ class ClusterCommand : public InteractionModelCommands, public ModelCommand, pub
                            const chip::app::Clusters::IcdManagement::Commands::RegisterClient::Type & value)
     {
         ReturnErrorOnFailure(InteractionModelCommands::SendCommand(device, endpointId, clusterId, commandId, value));
-        mScopedNodeId     = chip::ScopedNodeId(value.checkInNodeID, device->GetSecureSession().Value()->GetFabricIndex());
+        mScopedNodeId     = chip::ScopedNodeId(device->GetDeviceId(), device->GetSecureSession().Value()->GetFabricIndex());
```

Comment:
we need to use checkin nodeId instead of peer nodeId, since we may register different nodeId for LIT.

Reply:
In that case, what is stored during the commissioning registration is incorrect - the ICDs NodeId was being stored.

I'll revert this changes and fix the PairingCommand bits.

Reply:
yes, please fix commissioning registration and allow to set the other nodeId, thanks

Reply:
done